engine/src/main/java/org/terasology/rendering/opengl/FBOConfig.java

```diff
@@ -154,51 +139,38 @@ public FBOConfig useStencilBuffer() {
         return this;
     }
 
-    /**
-     * Given information set through the constructor and the use*Buffer() methods, builds and returns
-     * an FBO instance, inclusive the underlying OpenGL FrameBuffer and any requested attachments.
-     * <p>
-     * The FBO is also automatically registered with the LwjglRenderingProcess through its title string.
-     * This allows its retrieval and binding through methods such as getFBO(String title) and
-     * bindFBO(String title). If another FBO is registered with the same title, it is disposed and
-     * the new FBO registered in its place.
-     * <p>
-     * This method is effectively mono-use: calling it more than once will return the exact same FBO
-     * returned the first time. To build a new FBO with identical or different characteristics it's
-     * necessary to instantiate a new builder.
-     *
-     * @return An FBO. Make sure to check it with FBO.getStatus() before using it.
-     */
-    public FBO generate(FBO.Dimensions fullScale) {
-        // TODO: Investigate whether removing this would cause any problems
-        if (generatedFBO != null) {
-            return generatedFBO;
-        }
 
-        if (isRelativeToScreenSize) {
-            if (dimensions == null || dimensions.areDifferentFrom(fullScale)) {
-                dimensions = new FBO.Dimensions(fullScale);
-                dimensions.multiplySelfBy(scale);
-            }
-        }
+    // TODO: rename them in future
+    // TODO: add javadocs
+    public boolean isUseDepthBuffer() {
```

Comment:
I'd simply call these methods useDepthBuffer(), useNormalBuffer() and so on. But do we actually need them? Are they used somewhere?